components/retro-go/rg_display.c

```diff
@@ -140,7 +140,11 @@ static void spi_init(void)
     };
 
     const spi_device_interface_config_t devcfg = {
-        .clock_speed_hz = SPI_MASTER_FREQ_40M,  // 80Mhz causes glitches unfortunately
```

Comment:
The esp32 can also run at 80mhz, the reason it's set to 40mhz is because the display can't support 80mhz.
Are you sure it never glitches on your display?